lib/utils/mention.ts

```diff
@@ -39,14 +39,14 @@ function escape(value: string): string {
     //  2. Escape all non-escape control character
     return (
         value
-            .replace(/\\([\s\S])/g, (m, char) => {
+            .replace(/\\(?<char>[\s\S])/gu, (m, _1, _o, _s, { char }) => {
                 if (char.charCodeAt(0) < 0x20) {
                     return escapeControl(char)
                 }
                 return m
             })
-            // eslint-disable-next-line no-control-regex -- x
-            .replace(/[\0-\x1f]/g, escapeControl)
+            // eslint-disable-next-line no-control-regex, regexp/no-control-character -- x
```

Comment:
Why is `regexp/no-control-character` active? Are we using our `all` config?

Reply:
I agree that using `recommended` is better.

Reply:
I will change this PR once the `no-unused-capturing-group` issue is resolved :+1: